docs/docker-compose-ca.yml.example

```diff
@@ -0,0 +1,18 @@
+# Example docker-compose file for the SSH CA.
+version: '3.7'
+services:
+  kbsshca:
+    image: ca:latest
+    container_name: kbsshca
+    #command: ["./docker/entrypoint-server.sh"]
+    #command: ["./docker/entrypoint-generate.sh"]
+    command: ["sh", "-c", "sleep 1000"]
+    environment:
+      TEAMS: 'list,of,teams'
+      KEYBASE_USERNAME: 'yourusername'
+      KEYBASE_PAPERKEY: 'your paper key' # ideally, add this as a docker secret, and not in plaintext here
+      FORCE_WRITE: 'false'
+    volumes: # mounting the entrypoint scripts for easy iterative development.
```

Comment:
Should this be in the version we commit?

Reply:
removed